wsd/DocumentBroker.cpp

```diff
@@ -2743,7 +2744,8 @@ bool DocumentBroker::sendUnoSave(const std::shared_ptr<ClientSession>& session,
 
     // Note: It's odd to capture these here, but this function is used from ClientSession too.
     bool autosave = isAutosave || (_unitWsd && _unitWsd->isAutosave());
-    bool background = forceBackgroundSave || (autosave && _backgroundAutoSave);
```

Comment:
So - for the very last save of the document - we don't want to do a background save. Can you check from the logs that if you modify a document, then close the window without saving - that the final 'force' save is not a background save - even with the settings turned on.

Thanks!